DemoqaProject/pageObjects/Elements.cs

```diff
@@ -7,10 +7,323 @@ namespace DemoqaProject.PageObjects
+        public IWebElement textBoxButton;
+
+        [FindsBy(How = How.XPath, Using = "//input[@id='userName']")]
+        public IWebElement fullNameInput;
+
+        [FindsBy(How = How.XPath, Using = "//input[@id='userEmail']")]
+        public IWebElement emailInput;
+
+        [FindsBy(How = How.XPath, Using = "//textarea[@id='currentAddress']")]
+        public IWebElement currentAddressInput;
+
+        [FindsBy(How = How.XPath, Using = "//textarea[@id='permanentAddress']")]
+        public IWebElement permanentAddressInput;
+        
+        [FindsBy(How = How.XPath, Using = "//button[@id='submit']")]
+        public IWebElement submitButton;
+        
+        [FindsBy(How = How.XPath, Using = "//form[@id='userForm']")]
+        public IWebElement userForm;
+
+        [FindsBy(How = How.XPath, Using = "//p[@id='name']")]
+        public IWebElement fullnameText;
+        
+        [FindsBy(How = How.XPath, Using = "//p[@id='email']")]
+        public IWebElement emailText;
+        
+        [FindsBy(How = How.XPath, Using = "//p[@id='currentAddress']")]
+        public IWebElement currentAddressText;
+        
+        [FindsBy(How = How.XPath, Using = "//p[@id='permanentAddress']")]
+        public IWebElement permanentAddressText;
+
+        [FindsBy(How = How.XPath, Using = "//input[@class='mr-sm-2 field-error form-control']")]
+        public IWebElement errorClass;
+
+        [FindsBy(How = How.XPath, Using = "//span[text()='Check Box']")]
+        public IWebElement checkBoxButton;
+
+        [FindsBy(How = How.ClassName, Using = "rct-checkbox")]
+        public IWebElement selectAll;
+
+        [FindsBy(How = How.Id, Using = "result")]
+        public IWebElement checkSelectedResult;
+
+        [FindsBy(How = How.XPath, Using = "//button[@class='rct-collapse rct-collapse-btn'][1]")]
+        public IWebElement arrowButton;
+        
+        [FindsBy(How = How.XPath, Using = "//span[text()='Desktop']")]
+        public IWebElement desktopButton;
+        
+        [FindsBy(How = How.XPath, Using = "//span[text()='Commands']")]
+        public IWebElement commandsButton;
+        
+        [FindsBy(How = How.XPath, Using = "//button[@title='Expand all']")]
+        public IWebElement expendAll;
+        
+        [FindsBy(How = How.XPath, Using = "//button[@title='Collapse all']")]
+        public IWebElement collapseAll;
+        
+        [FindsBy(How = How.XPath, Using = "//span[text()='Radio Button']")]
+        public IWebElement radioButton;
+        
+        [FindsBy(How = How.XPath, Using = "//label[@for='yesRadio']")]
+        public IWebElement yesRadio;
+        
+        [FindsBy(How = How.XPath, Using = "//label[@for='impressiveRadio']")]
+        public IWebElement impressiveRadio;
+        
+        [FindsBy(How = How.XPath, Using = "//label[@for='noRadio']")]
+        public IWebElement noRadio;
+        
+        [FindsBy(How = How.ClassName, Using = "mt-3")]
```

Comment:
Such class locators are pretty unstable for elements locator use, try to avoid using them.

Reply:
sure